Tests/Test Helpers/TestImage.cs

```diff
@@ -0,0 +1,13 @@
+﻿using UnityEngine.UI;
+
+// if you create a custom class in a file that doesn't match the class name,
+// after the tests run Unity will complain that the compiler that imported 
+// "TestImage" isn't available anymore, so they go in their own files
+namespace UnityEditor.Experimental.EditorVR.Tests.TestHelpers
+{
+    public class TestImage : Graphic
```

Comment:
i can just remove the comment - the class exists in a separate file just to make Unity not complain about it